qcodes/instrument_drivers/tektronix/Keithley_2600_channels.py

```diff
@@ -79,6 +81,102 @@ def get_raw(self) -> np.ndarray:
         return data
 
 
+class TimeTrace(ParameterWithSetpoints):
+    """
+    A parameter class that holds the data corresponding to the time dependence of
+    current and voltage.
+    """
+
+    def __init__(self, name: str, instrument: Instrument, **kwargs) -> None:
+
+        super().__init__(name=name,
+                         instrument=instrument,
+                         **kwargs)
+
+    def _prepareTimeTrace(self, mode: str) -> None:
```

Comment:
Is there still a reason for this function to take a `mode` arg?

As far as I can see, this function is now a (weak) validation function; it either does nothing or gives a warning. That is not a bad thing to have, but the name and docstring should reflect the usage.

Reply:
Hi, I changed the name to _validateTimeTrace. It needs mode argument as indeed it checks the mode and raises a ValueError. These are written in the docstring as well.